packages/ezjs_min/ezjs_min.0.2.2/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "A bunch of js_of_ocaml shortcuts"
+description: "A bunch of js_of_ocaml shortcuts"
+maintainer: "OCamlPro <contact@ocamlpro.com>"
+authors: "OCamlPro <contact@ocamlpro.com>"
+license: "LGPL-2.1"
+homepage: "https://github.com/ocamlpro/ezjs_min"
+bug-reports: "https://github.com/ocamlpro/ezjs_min/issues"
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {>= "2.8" & >= "2.8"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```

You can remove dune as a dependency in the dune-project file, it should be automatically added to the generated opam file